e2e/vite/src/vite.test.ts

```diff
@@ -144,6 +146,24 @@ describe('Vite Plugin', () => {
         expect(fileExists(`dist/apps/${myApp}/package.json`)).toBeFalsy();
         rmDist();
       }, 200_000);
+
+      it('qqqqq should build application with package json generation', async () => {
```

Comment:
Remove the `qqqqq` part, and we should test with and without existing `package.json` file.